src/scripts/site.js

```diff
@@ -620,514 +589,6 @@ var AppInfo = {};
         });
     }
 
-    function defineRoute(newRoute, dictionary) {
-        var baseRoute = Emby.Page.baseUrl();
-        var path = newRoute.path;
-        path = path.replace(baseRoute, "");
-        console.log("Defining route: " + path);
-        newRoute.dictionary = newRoute.dictionary || dictionary || "core";
-        Emby.Page.addRoute(path, newRoute);
-    }
-
-    function defineCoreRoutes(appHost) {
-        console.log("Defining core routes");
-        defineRoute({
-            path: "/addplugin.html",
-            dependencies: [],
-            autoFocus: false,
-            roles: "admin",
-            controller: "scripts/addpluginpage"
-        });
-        defineRoute({
-            path: "/autoorganizelog.html",
-            dependencies: [],
-            roles: "admin"
-        });
-        defineRoute({
-            path: "/channelsettings.html",
-            dependencies: [],
-            autoFocus: false,
-            roles: "admin"
-        });
-        defineRoute({
-            path: "/addserver.html",
-            dependencies: ["emby-button", "emby-input"],
```

Comment:
why dependencies are gone in new code?

Reply:
They aren't. They are all required in the start of the module routes.js. The effect is the same, since if you manage to load all pages, the result would be requiring all modules present in dependencies. No specific usage for then in routes, just requiring is necessary.